### PR TITLE
EOL for QtQuick1, enable HiDPI auto-scaling

### PR DESCRIPTION
**Pull request by nirvn**
@m-kuhn , et voila, we're there, last step. I've rebased against master to only keep the last remaining commits in this PR.

A big chunk of this is only upgrading / harmonizing the import version numbers.

### CODE REVIEW

File: src/qgsquick/qgsquickmapcanvasmap.cpp [hunk 1]
```diff
@@ -33,6 +33,8 @@ QgsQuickMapCanvasMap::QgsQuickMapCanvasMap( QQuickItem *parent )
   : QQuickItem( parent )
   , mMapSettings( qgis::make_unique<QgsQuickMapSettings>() )
 {
+  mMapSettings->setOutputDpi( 96 );
```
Comment (m-kuhn) [inline src/qgsquick/qgsquickmapcanvasmap.cpp:36]: Fixed?

Comment (nirvn) [reply src/qgsquick/qgsquickmapcanvasmap.cpp:36]: Well spotted, this isn't needed.

File: src/qml/FeatureListForm.qml [hunk 2]
```diff
@@ -15,12 +15,15 @@
...
-import QtQuick.Layouts 1.1
+import QtQuick 2.12
+import QtQuick.Controls 2.12
+import QtQuick.Layouts 1.12
 import QtQuick.Dialogs 1.2
+
 import org.qgis 1.0
 import org.qfield 1.0
 import Theme 1.0
+import "."
```
Comment (m-kuhn) [inline src/qml/FeatureListForm.qml:26]: What's that one for?

Comment (nirvn) [reply src/qml/FeatureListForm.qml:26]: Button { id: deleteButton... }

Comment (m-kuhn) [reply src/qml/FeatureListForm.qml:26]: Next step: kill all those "." imports and put those into a library like Theme

Comment (nirvn) [reply src/qml/FeatureListForm.qml:26]: @m-kuhn , I've actually tried, and couldn't get it to compile then. Might have just been me juggling with too many PRs.

**Comment by nirvn** [comment]
We're literally 10 minutes away from leaving QtQuick 1 behind. 

:drum: :drum: :drum:

**Comment by m-kuhn** [comment]
I'll cheers to that with a nice glass of water.

**Comment by qfield-fairy** [comment]
Uploaded test apks for
 - [**armv7**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-991-38573ab9bed8d3276dc1146a6e509235ebbdb5e7-armv7.apk)
 - [**x86**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-991-38573ab9bed8d3276dc1146a6e509235ebbdb5e7-x86.apk)
 - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-991-38573ab9bed8d3276dc1146a6e509235ebbdb5e7-arm64_v8a.apk)
 - [**x86_64**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-991-38573ab9bed8d3276dc1146a6e509235ebbdb5e7-x86_64.apk)

**Comment by m-kuhn** [comment]
:milk_glass:

**Pull request merged**